include/z64save.h

```diff
@@ -79,7 +79,7 @@ typedef struct PermanentSceneFlags {
     /* 0x0C */ u32 clearedRoom;
     /* 0x10 */ u32 collectible;
     /* 0x14 */ u32 unk_14;
```

Comment:
```suggestion
    /* 0x14 */ u32 unk_14; // varies based on scene. For dungeons, floors visited. 
```
Maybe we can record what we worked out? Optional though